spack_configs/blueos_3_ppc64le_ib/packages.yaml

```diff
@@ -12,9 +12,9 @@ packages:
     buildable: False
 
   cuda:
-    version: [10.1.243,10.1.168,9.2.148,8.0]
+    version: [10.1.168,11.0.2,9.2.148,8.0]
     paths:
-      cuda@10.1.243: /usr/tce/packages/cuda/cuda-10.1.243
+      cuda@11.0.2: /usr/tce/packages/cuda/cuda-11.0.2
```

Comment:
10.1.243 is what RAJA is using, 10.1.168 is what Umpire is using.
Can you confirm that you want RAJA to move to 10.1.168?

Reply:
I'll keep both.